spec/models/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+
+
+describe User, type: :model do
+  describe "validation" do
+    it {should validate_presence_of(:first_name)}
+    it {should validate_presence_of(:last_name)}
+    it {should validate_presence_of(:email)}
+    it {should validate_presence_of(:password)}
```

Comment:
Looks like these tests aren't passing on TravisCI, do we still need to migrate? Or is shoulda-matchers not working?

Reply:
Actually It looks like we need to add the shoulda-matchers config to the rails-helper.

Reply:
Just fixed it, was missing a few lines in the rails_spec

Reply:
It looks like the shoulda-matchers configure block isn't in rails_helper yet. I'll drop it below and @EricLarson2020 you can insert it and run the tests?